test/t/matches/timeslots_test.exs

```diff
@@ -293,6 +293,73 @@ defmodule T.Matches.TimeslotsTest do
     test "timeslot_reminder not scheduled for slots within 15 minutes from now"
   end
 
+  describe "accept_now_slot/2 side-effects" do
+    setup [:with_profiles, :with_match, :with_now_offer]
+
+    setup %{profiles: [p1, p2], match: match} do
+      :ok = Matches.subscribe_for_user(p1.user_id)
+
+      %Timeslot{} =
+        Matches.accept_slot_for_match(
+          p2.user_id,
+          match.id,
+          _slot = "2021-03-23 14:15:00Z",
+          _reference = ~U[2021-03-23 14:25:00Z]
+        )
+
+      :ok
+    end
+
+    test "accept broadcasted via pubsub to mate" do
+      assert_receive {Matches, [:timeslot, :accepted], %Timeslot{} = timeslot}
+
+      assert timeslot.slots == [~U[2021-03-23 14:15:00Z]]
+
+      assert timeslot.match_id
+      assert timeslot.selected_slot == ~U[2021-03-23 14:15:00Z]
+    end
+
+    test "push notifications are scheduled", %{
```

Comment:
I'd put this test into `accept_slot/2 side-effects` block and change the name to describe that the test is for notifications of a timeslot that has already started

Reply:
then two different slots would be needed to be accepted there, I think it will be more clear if we keep these tests splitted

Reply:
> then two different slots would be needed to be accepted there

do you mean the setup block?

Reply:
yes

Reply:
It's not a problem, you'd remove slot offering from the setup block and only keep subscription there. There is no benefit either though since now `accept_slot/2 when slot in future side-effects` contains two tests.